### PR TITLE
Create mattermost-mobile-performance.json

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Add mobile performance dashboard

#### Ticket Link
Fix https://mattermost.atlassian.net/browse/MM-59858

### CODE REVIEW

File: grafana/mattermost-mobile-performance.json [hunk 1]
```diff
@@ -0,0 +1,2062 @@
...
+      },
+      "id": 13,
+      "panels": [],
+      "title": "Load",
+      "type": "row"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "P27C405C01959D762"
```
Comment (agnivade) [inline grafana/mattermost-mobile-performance.json]: Datasources need to be templatized. See other examples: https://github.com/mattermost/mattermost-performance-assets/blob/0109a80aaf5fabb3b34ad7102347f8684ddbbfc8/grafana/mattermost-web-app-metrics.json#L78

**Review by agnivade — CHANGES_REQUESTED** [review]

**Review by agnivade — CHANGES_REQUESTED** [review]
Please remove any mention to namespace. This is an internal only label and won't be used by our customers.

This would diverge from the internal dashboard we have. But unfortunately, there's no better way.

File: grafana/mattermost-mobile-performance.json [hunk 2]
```diff
@@ -0,0 +1,2081 @@
...
+  },
+  "time": {
+    "from": "now-24h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "browser",
+  "title": "Mobile Performance Metrics",
+  "uid": "ddp4txrlmyg3kc",
+  "version": 26,
```
Comment (devinbinnie) [inline grafana/mattermost-mobile-performance.json:2079]: I don't know if this version number matters or not, but at least keep in mind we should probably keep incrementing it? I guess it's just a build number :P

**Review by devinbinnie — APPROVED** [review]

File: grafana/mattermost-mobile-performance.json (hunk 2 above)
Comment (larkox) [reply grafana/mattermost-mobile-performance.json:2079]: I guess every time we hit save in the dashboard, it increases. 0/5 on what is the plan for this here, though.

**Review by agnivade — APPROVED** [review]
Thanks!

File: grafana/mattermost-mobile-performance.json (hunk 2 above)
Comment (agnivade) [reply grafana/mattermost-mobile-performance.json:2079]: It's a good practice to keep it increasing to keep track of changes. But not a huge blocker.

**Pull request merged**